src/web-auth/captcha.js

```diff
@@ -323,11 +325,9 @@ function runArkose() {
  * @param {Function} [options.templates.recaptcha_enterprise] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.hcaptcha] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.friendly_captcha] template function receiving the challenge and returning a string
+ * @param {Function} [options.templates.arkose] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.error] template function returning a custom error message when the challenge could not be fetched, receives the error as first argument
  * @param {String} [options.lang=en] the ISO code of the language for recaptcha
- * @param {Object} [options.callbacks] An optional object containing callbacks called after captcha events (only for Arkose captcha provider)
- * @param {Function} [options.callbacks.onSolved] An optional callback called after the captcha is solved (only for Arkose captcha provider)
- * @param {Function} [options.callbacks.onError] An optional callback called after the captcha encounters an error with the error passed as the first argument (only for Arkose captcha provider)
  * @param {Function} [callback] An optional callback called after captcha is loaded
  * @ignore
```

Comment:
Why isn't the return value documented here? 🤔 It would be nice to have some docs for the 4 returned methods, no? Or am I missing it?

In particular, we could mention in the docs that `requiresTrigger()` will only return a valid value after the captcha has been loaded (ie. after `callback` has been called). Similar, that `triggerCaptcha()` should also be called only after the captcha has been loaded.

Reply:
@frederikprijck Is there a reason why most of the JS Doc doesn't have `@returns` in here? I'm gonna add a `@returns` for the captcha functions.

Reply:
I do not think there is a specific reason, I believe alot of the API's do not return anything, but are callback based.

Reply:
Let's remove the added documentation here. A return value is not something we want, nor should promote in a callback based API.

Reply:
To circle back, we should not document this as we should not have a return value in a callback based API to begin with. We can not drop this, but let's also not extend it any further but properly implement callback based API's going forward.